DEV/alter_crops_obic.R

```diff
@@ -437,9 +457,12 @@ cr <- cr[,V1 := NULL]
   cr[is.na(crop_season), crop_season := 'overig']
 
   # Correction on Spelt waterstress
-  cr[crop_name == 'spelt', crop_waterstress := 'overig']
+  cr[crop_name == 'spelt', crop_waterstress := 'granen']
 
   # update the csv and Rdata file
+  # replace crop names with original capitalised names
+  cr[,crop_name := NULL]
```

Comment:
why this. i think everying in lower case is much easier to avoid errors later on.
so, i would not recommend to change this back.
if used in fuctions (i am not sure if that is done, i think mostly not) then adapt these to lower case too.